components/src/preact/map/sequences-by-location.tsx

```diff
@@ -0,0 +1,140 @@
+        </ErrorBoundary>
+    );
+};
+
+const SequencesByLocationMapInner: FunctionComponent<SequencesByLocationProps> = (props) => {
+    const { lapisFilter, lapisLocationField, mapSource } = props;
+
+    const lapis = useContext(LapisUrlContext);
+    const { isLoading: isLoadingMap, geojsonData } = useGeoJsonMap(mapSource);
+    const {
+        data,
+        error,
+        isLoading: isLoadingLapisData,
+    } = useQuery(
+        async () => queryAggregateData(lapisFilter, [lapisLocationField], lapis),
+        [lapisFilter, lapisLocationField, lapis],
+    );
+
+    if (isLoadingMap || isLoadingLapisData) {
+        return <LoadingDisplay />;
+    }
+
+    if (error) {
+        throw error;
+    }
+
+    return <SequencesByLocationMapTabs geojsonData={geojsonData} data={data} originalComponentProps={props} />;
+};
+
+type SequencesByLocationMapTabsProps = {
+    originalComponentProps: SequencesByLocationProps;
+    geojsonData: GeoJSON.FeatureCollection<GeometryObject, GeoJsonFeatureProperties>;
+    data: AggregateData;
+};
+
+const SequencesByLocationMapTabs: FunctionComponent<SequencesByLocationMapTabsProps> = ({
+    originalComponentProps,
+    geojsonData,
+    data,
+}) => {
+    const getTab = (view: SequencesByLocationMapView) => {
+        switch (view) {
+            case views.map:
+                return {
+                    title: 'Map',
+                    content: (
+                        <SequencesByLocationMap
+                            locationData={data}
+                            geojsonData={geojsonData}
+                            enableMapNavigation={originalComponentProps.enableMapNavigation}
+                            lapisLocationField={originalComponentProps.lapisLocationField}
+                            zoom={originalComponentProps.zoom}
+                            offsetX={originalComponentProps.offsetX}
+                            offsetY={originalComponentProps.offsetY}
+                        />
+                    ),
+                };
+        }
+    };
+
+    const tabs = originalComponentProps.views.map((view) => getTab(view));
+
+    return <Tabs tabs={tabs} toolbar={<Toolbar originalComponentProps={originalComponentProps} />} />;
+};
+
+type ToolbarProps = {
+    originalComponentProps: SequencesByLocationProps;
+};
+
+const Toolbar: FunctionComponent<ToolbarProps> = ({ originalComponentProps }) => {
+    return (
+        <div class='flex flex-row'>
+            <SequencesByLocationMapInfo originalComponentProps={originalComponentProps} />
+            <Fullscreen />
+        </div>
+    );
+};
+
+type SequencesByLocationMapInfoProps = {
+    originalComponentProps: SequencesByLocationProps;
+};
+
+const SequencesByLocationMapInfo: FunctionComponent<SequencesByLocationMapInfoProps> = ({ originalComponentProps }) => {
+    const lapis = useContext(LapisUrlContext);
+    return (
+        <Info>
+            <InfoHeadline1>Prevalence By Location</InfoHeadline1>
```

Comment:
```suggestion
            <InfoHeadline1>Sequence by location</InfoHeadline1>
```
So far we have only capitalized the first letter.